net/README.md

```diff
@@ -31,11 +31,11 @@ NOTE: This example uses GCE.  If you are using AWS EC2, replace `./gce.sh` with
 
 ```bash
 $ cd net/
-$ ./gce.sh create -n 5 -c 1  #<-- Create a GCE testnet with 5 additional nodes (beyond the bootstrap node), 1 client bencher (billing starts here)
+$ ./gce.sh create -n 5 -c 1     #<-- Create a GCE testnet with 5 additional nodes (beyond the bootstrap node) and 1 client (billing starts here)
 $ ./init-metrics.sh $(whoami)   #<-- Configure a metrics database for the testnet
-$ ./net.sh start             #<-- Deploy the network from the local workspace
-$ ./ssh.sh                   #<-- Details on how to ssh into any testnet node to access logs/etc
-$ ./gce.sh delete            #<-- Dispose of the network (billing stops here)
+$ ./net.sh start -c 1           #<-- Deploy the network from the local workspace with one solana-bench-tps client
```

Comment:
I was thinking without `-c 1` (ie, no `-c` specified at all), `./net.sh start` would default to `-c bench-tps=X` where `-c X` was supplied to `./gce create`.   This keeps us fully compatible and we don't have to go modify all the `ci/` downstream scripts that use `net/`

Reply:
Yeah, that is how it works, checkout net.sh.  I can update the README.md to show the default, non specifying case

Reply:
Opps I missed that.

> I can update the README.md to show the default, non specifying case

sg!